importers/src/bible/sword/src/importer.ts

```diff
@@ -32,6 +32,7 @@ export class SwordImporter extends BibleEngineImporter {
                 language: 'en-US',
                 chapterVerseSeparator: ':',
                 hasStrongs: true,
+                isPlaintext: true,
```

Comment:
unfortunately, the ESV sword module doesn't always wrap phrases in an enclosing group. 

Neither does the CUV sword module. This is likely true for all sword modules.

Because of this, I set the flag here, to avoid an import error in Deuteronomy.

Reply:
i think we have a general conceptual decision here. i would prefer if one could expect a very stable output format out of bible engine. that would mean to be strict during input. the way i do it is to offload complexity onto the importer so that we don't need hacks from thereon.

concerning paragraphs this is a conceptual decision to say that everything needs to be inside a paragraph (or nothing, then the renderer can decide to either group by verse, by chapter or not at all). we can discuss this decision, e.g. one could argue that there can be other paragraph types, like "line group". however that makes things more complex in the renderer and - since we are dealing with bible content - in many other use cases, since having a consistent way of grouping together bible texts on a semantic level enables multiple meaningful use cases. thats why bible engine also has a dedicated table for paragraphs.

so my suggestion would be to fix/add missing paragraphs in the importer.

if you use the `isPlaintext` hack this will lead to other issues. the question is, what should UI-engineers expect from the bible engine output? they could decide to render the content by paragraph - in that case there would be text missing in the render. or they could decide to skip on all formatting since the `isPlaintext` flag is set (which would be a reasonable thing to do). So if we want to enable the text-outside-paragraph thing in formatted bibles (which i hope we can avoid), then there should be another flag like `canHaveTextOutsideParagraphs`

Reply:
I can update the importer to always wrap phrases in a paragraph, I guess thats ok. 

Added a ticket for it: https://github.com/tyndale/BibleEngine/issues/152

Reply:
Don't think I can resolve this issue in this PR, which is already getting massive 😅 

But I've added the followup task to the project board

It makes sense what you're saying. Just need time to dig into the importer.